### ISSUE TITLE
use R3.3.0 and im2rec.py to run cnn and always failed

### ISSUE THREAD

**Issue by janejane11** [apache/incubator-mxnet#7262]
this is my code below,I have run it many times,but it always failed and my computer stopped working, I check my memories,my memories are 8GB
it failed because my memories are not enough? or some other problems?
please answer me how can I run it successfully

library(mxnet)

train = mx.io.ImageRecordIter(
  path.imglist="C:/train1/train1.lst",
  path.imgrec="C:/train1/train1.rec",
  batch.size=4,
  #round.batch=TRUE,
  data.shape=c(224,224,1),
  preprocess.threads=1
)

test = mx.io.ImageRecordIter(
  path.imglist="C:/test1/test1.lst",
  path.imgrec="C:/test1/test1.rec",
  batch.size=4,
  data.shape=c(224,224,1),
  #round.batch=FALSE,
  preprocess.threads=1
)

data <- mx.symbol.Variable('data')

conv1 <- mx.symbol.Convolution(data=data, kernel=c(5,5), num_filter=20)
tanh1 <- mx.symbol.Activation(data=conv1, act_type="tanh")
pool1 <- mx.symbol.Pooling(data=tanh1, pool_type="max",kernel=c(2,2), stride=c(2,2))

conv2 <- mx.symbol.Convolution(data=pool1, kernel=c(5,5), num_filter=50)

tanh2 <- mx.symbol.Activation(data=conv2, act_type="tanh")
pool2 <- mx.symbol.Pooling(data=tanh2, pool_type="max",kernel=c(2,2), stride=c(2,2))

flatten <- mx.symbol.Flatten(data=pool2)
fc1 <- mx.symbol.FullyConnected(data=flatten, num_hidden=500)
tanh3 <- mx.symbol.Activation(data=fc1, act_type="tanh")

fc2 <- mx.symbol.FullyConnected(data=tanh3, num_hidden=10)

lenet <- mx.symbol.SoftmaxOutput(data=fc2)

mx.set.seed(0)

model <- mx.model.FeedForward.create(lenet, X=train, y=train.y,
ctx=mx.cpu(), num.round=5, array.batch.size=1,
learning.rate=0.05, momentum=0.9, wd=0.00001,
eval.metric=mx.metric.accuracy,
epoch.end.callback=mx.callback.log.train.metric(100))

preds <- predict(model, test.array)

pred.label <- max.col(t(preds)) - 1

table(test_org[,1],pred.label)

sum(diag(table(test_org[,1],pred.label)))/1000
--------------------------------------------------------------------end

**Comment by thirdwing** [comment]
Can you tell us how installed the mxnet and the version of mxnet?

Please also show us the error message and share your data.

**Comment by ankkhedia** [comment]
@janejane11 Thanks for trying out MXNetR. 
Sorry for getting back to you after little while but wanted to know if you are still facing the issue. Also, could you please share the details(error log /dataset) to help you better on solving the issue.

**Comment by ankkhedia** [comment]
@nswamy @sandeep-krishnamurthy Could you please label the issue as Pending Requester Info

**Comment by ankkhedia** [comment]
@sandeep-krishnamurthy @nswamy Could you please close this issue due to inactivity

@janejane11  Please feel free to reopen if closed in error
